temporal.go

```diff
@@ -269,17 +269,17 @@ func (duration FEELDuration) MarshalJSON() ([]byte, error) {
 
 func (duration FEELDuration) Duration() time.Duration {
```

Comment:
Correct me if I am wrong but we can calculate the time.Duration only when we know the date to which feel duration from iso 8601 applies to. And even then we should use functions AddDate and Add to calculate it correctly.

Reply:
Uugh, good question ... need to dig deeper on that.
Did create a ticket for that to be re-evaluated: https://github.com/pbinitiative/feel/issues/16
Prefer to tackle this separately.